### DISCUSSION TITLE
Figured out the other file where they store the miscellaneous UI strings

### DISCUSSION THREAD

**Discussion by ToastyBuns3939** [chinosk6/scsp-localify#15] [Show and tell]
Looking through the game files, it looks like they stored some UI strings like the idol profiles in this file.

![image](https://github.com/chinosk6/scsp-localify/assets/93896872/95122a3f-425f-4bbf-8094-de289ef296b7)

[masterdata.bytes](https://drive.google.com/file/d/1WUaPfhhiE4IekYctYy1J9Y7ysVq4YWvK/view?usp=sharing)

a quick google suggested that these are `Unity Binary Text Asset Data` files

edit: the file also contains the list of audio files used in each commus. apparently they stored the character names for each commu line here too aside from the actual audio files themselves

![image](https://github.com/chinosk6/scsp-localify/assets/93896872/e90f71ff-94e0-40bd-a53e-1ff1fa4e3f10)

**Comment by chinosk6** [comment]
It's a [MemoryPack](https://github.com/Cysharp/MemoryPack) file, serialized from `PRISM.Definitions.MasterData` class.
The following is the class serialized using the `JsonConvert.SerializeObject` method
[MasterData.json](https://github.com/chinosk6/scsp-localify/files/14864894/master_dump.json)
